helios-services/src/main/java/com/spotify/helios/servicescommon/ServiceParser.java

```diff
@@ -185,6 +186,11 @@ public ServiceParser(final String programName, final String description, final S
         .setDefault(new ArrayList<String>())
         .help("Kafka brokers to bootstrap with");
 
+    pubsubTopicPrefixArg = parser.addArgument("--pubsub-topic-prefix")
+        .action(append())
+        .setDefault(new ArrayList<String>())
+        .help("Google pubsub topic prefixes to send to");
```

Comment:
is the idea that the user will specify 0 or more prefixes like "foobar.helios." and then the helios agent or master will provide the rest of the topic name, like `"foobar.helios." + "HeliosTaskStatusEvents"`?

It would probably be helpful to document somewhere the full list of topics that the helios codebase will send messages to (when enabled)